### ISSUE TITLE
setup raven.js

### ISSUE THREAD

**Issue by xcambar** [dschmidt/ember-cli-deploy-sentry#22]
From the README:

> automatically setup raven-js? If you want this, let me know.

I think this would be a great addition to the addon as it would allow a simpler setup and overall a better experience of Sentry+Ember.

If it appears relevant to the community, I can guess two options:
- merge `ember-cli-deploy-sentry` and `ember-cli-sentry`
- use `ember-cli-sentry` as a dependency for `ember-cli-deploy-sentry`

I'd be more than happy to discuss the options.

**Comment by dschmidt** [comment]
Second option is basically working already.. ember-cli-sentry and ember-cli-deploy-sentry work along just fine together, it's just that ember-cli-sentry is not a required depdendency

**Comment by xcambar** [comment]
Yes, I understand, yet my point was to highlight that some/all of the apps that use `ember-cli-deploy-sentry` will most likely want to use `ember-cli-sentry`, hence it would make sense that `ember-cli-deploy-sentry` provides and possibly bundles some defaults for `ember-cli-sentry`.

**Comment by RobinDaugherty** [comment]
The two projects work side-by-side, but do not work together without further configuration. This project sets the application revision in a meta tag, but ember-cli-sentry does not use the meta tag, and sends reports with a different revision. I think this worked in the past, but it does not work today.

**Comment by duizendnegen** [comment]
@RobinDaugherty check out https://github.com/dschmidt/ember-cli-deploy-sentry/issues/39 as a proposal to get rid of the meta tag generally.
